### PR TITLE
Update additional params from KubernetesManager

### LINKED ISSUE #132: Allow export of access method information for applications deployed in bulk
When using the bulk application deployment functionality, it should be possible to export access information about the deployed applications using a CSV file.

The CSV file should contain the following information:

- username
- domain
- application name
- application instance name
- application version
- values of any customized parameters (each in a separate column)
- information about all supported access methods by the application (HTTP, SSH...)

### PR DESCRIPTION
**Pull request by kbeyro**

### CODE REVIEW

File: src/main/java/net/geant/nmaas/nmservice/deployment/containerorchestrators/kubernetes/KubernetesManager.java [hunk 1]
```diff
@@ -447,6 +449,17 @@ public AppUiAccessDetails serviceAccessDetails(Identifier deploymentId) {
         }
     }
 
+    @Override
+    public Map<String, String> retrieveDeployParameters(Identifier deploymentId) {
+        try {
+            retrieveOrUpdateInternalServiceIpAddress(repositoryManager.loadService(deploymentId));
```
Comment (llopat) [inline src/main/java/net/geant/nmaas/nmservice/deployment/containerorchestrators/kubernetes/KubernetesManager.java]: Is this call relevant here?

File: src/main/java/net/geant/nmaas/portal/service/impl/BulkApplicationServiceImpl.java [hunk 2]
```diff
@@ -342,19 +342,13 @@ public List<BulkAppDetails> getAppsBulkDetails(BulkDeploymentView bulkDeployment
             AppInstance instance = instanceService.find(instanceId).orElseThrow();
 
             Map<String, String> configurationParameters = new HashMap<>();
-            if (!Objects.isNull(instance.getConfiguration())) {
```
Comment (llopat) [inline src/main/java/net/geant/nmaas/portal/service/impl/BulkApplicationServiceImpl.java:345]: So we're not adding those anymore?

Comment (kbeyro) [reply src/main/java/net/geant/nmaas/portal/service/impl/BulkApplicationServiceImpl.java:345]: Additional params contains configuration params

File: src/main/java/net/geant/nmaas/nmservice/deployment/containerorchestrators/kubernetes/KubernetesManager.java (hunk 1 above)
Comment (kbeyro) [reply src/main/java/net/geant/nmaas/nmservice/deployment/containerorchestrators/kubernetes/KubernetesManager.java]: To be deleted

**Review by llopat — APPROVED** [review]

**Pull request merged**